### PR TITLE
Update french translation for 4.3

### PR DESCRIPTION
**Pull request by damienbigot**
- Completely rewriting french translation for better consistency with Adobe products (in their french version).
- Adding translation for new features.

### CODE REVIEW

File: src/messages.coffee [hunk 1]
```diff
@@ -553,8 +553,9 @@ define ->
           usage information to help improve the product?
         "
         fr_fr: -> "
-          Autorisez-vous GuideGuide à utiliser Google Analytics pour suivre
-		  anonymement les informations d'utilisation pour améliorer le plugin ?
+          Autoriser GuideGuide à utiliser Google Analytics pour collecter 
+          anonymement des informations d'utilisation et aider à améliorer le 
+          plugin ?
```
Comment (cameronmcefee) [inline src/messages.coffee]: > plugin ?

Is the space before the `?` a part of French grammar, or a typo?

**Comment by cameronmcefee** [comment]
I added one question inline, otherwise this looks great. Thanks!

File: src/messages.coffee (hunk 1 above)
Comment (damienbigot) [inline src/messages.coffee]: Yes, many punctuations have a space before, like **_? ; : !_**
Some others don't, like **_,_** or **_._**

Comment (cameronmcefee) [inline src/messages.coffee]: Ok, great. Thanks for confirming.

**Comment by cameronmcefee** [comment]
:sparkles: thanks again!

**Pull request merged**